### ISSUE TITLE
Suggested feature: share selected text to Sesame

### ISSUE THREAD

**Issue by ghost** [BranchMetrics/sesame_issues#105]
Would it be possible to configure Sesame so that it could be a target when I select text in an app and share it? This would make it easier to turn text from an email or other source into a search.

**Comment by philwall2** [comment]
Good idea. Probably a lower priority (you can see the rest of the list). But a good idea none the less. Thanks!
